scripts/mppc_to_multiprobe.py

```diff
@@ -250,8 +261,8 @@ def main(args):
     parser.add_argument('--file', dest="file",
                         help="Image file to run the mppc to multiprobe alignment for.")
     parser.add_argument('--angle', dest="phi", required=True,
```

Comment:
Add `type=float` to force it to be a float (and then you won't have to convert it to float).